docs/stacks-guide-aws.md

```diff
@@ -0,0 +1,911 @@
+---
+title: "Crossplane Stacks Guide: AWS Setup" toc: true weight: 332 indent: true
+---
+
+# Crossplane Stacks Guide: AWS Setup
+
+## Table of Contents
+
+  - [Crossplane Stacks Guide: AWS Setup](#crossplane-stacks-guide-aws-setup)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Before you begin](#before-you-begin)
+  - [Install AWS stack](#install-aws-stack)
+  - [Configure AWS account](#configure-aws-account)
+  - [Configure Crossplane Provider for
+    AWS](#configure-crossplane-provider-for-aws)
+  - [Set Up Network Configuration](#set-up-network-configuration)
+  - [Configure Provider Resources](#configure-provider-resources)
+  - [Recap](#recap)
+
+## Introduction
+In this guide we are focusing on WordPress workload, and will show how it could
```

Comment:
I think using the term workload here is confusing because we have an actual Wordpress workload example. How would you feel about changing terminology to "stack"?